### DISCUSSION TITLE
[REQUEST] Pterodactyl game server manager

### DISCUSSION THREAD

**Discussion by l3gitpanda** [tteck/Proxmox#539] [Ideas]
Main website/docs:
https://pterodactyl.io/
Panel GitHub:
https://github.com/pterodactyl/panel
Wings(Client/node) GitHub:
https://pterodactyl.io/wings/1.0/installing.html

Really awesome game server manager you can use for any game, minecraft and steam cmd games included. Easy to use web interface and powerful management

**Comment by dreulavelle** [comment]
Being the largest game server administration for a large collection of games.. I also second this!

**Comment by DomJeal** [comment]
My instance certainly was a "fun" setup for a Linux noob like me to do in an LXC, so yeah a script would be nice.

**Comment by tteck** [comment]
At least there's https://www.turnkeylinux.org/gameserver that can be created with the TurnKey script.

**Reply by Data-PhiIe** [reply]
Unfortunately turnkeys is not nearly as feature rich, nor does it cover nearly the same broad scope of games that pterodactyl does. :( I ended up moving away from turnkeys solution, I actually ended up using the script in the comment below in a VM.

**Comment by petervk** [comment]
Another request for this. Maybe you can use this script to help automate the install? https://github.com/pterodactyl-installer/pterodactyl-installer

**Comment by BoringKraken** [comment]
I'm a complete Proxmox and Linux noob , something like this would be amazing to have.

currently trying to get pterodactyl of my unraid and onto my cluster, so I can set up a couple of nodes/wings.

**Comment by nicwn** [comment]
Another vote for this in 2024

**Comment by vrozaksen** [comment]
+1 But! I found they are working on something new like Pelican https://pelican.dev/

**Comment by exrith** [comment]
Would be great to have this or pelican

**Reply by Data-PhiIe** [reply]
better yet having an option in the setup process to select one or the other would be nice.

**Comment by Data-PhiIe** [comment]
For high availability's sake, having it as a VM over LXC would probably be better too, so It can live migrate seamlessly. LXC's have a very brief interruption when migrating between nodes. Not ideal for a game server where you might not want those interruptions.

**Comment by KemMuammer** [comment]
This would be great alltho it is the last version and Pelican https://pelican.dev/ will replace it so maybe go with that one
